trk/src/Components/ClimberProfile.js

```diff
@@ -40,9 +41,10 @@ const ClimberProfile = ({ navigation }) => {
             const newClimbsHistory = climbsSnapshots.map((climbSnapshot, index) => {
                 if (!climbSnapshot.exists) return null;
                 return { ...climbSnapshot.data(), tapId: filteredTaps[index].id };
-            }).filter(climb => climb !== null);
+            }).filter(climb => climb !== null && (climb.archived === undefined || climb.archived === false));
```

Comment:
I'm thinking that one day we should declare variables that cover all these checks at once.

E.g. safe = `climb !== null && (climb.archived === undefined || climb.archived === false))`